src/components/dls/Separator/Separator.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames';
+import * as SeparatorPrimitive from '@radix-ui/react-separator';
+import styles from './Separator.module.scss';
+
+export enum SeparatorOrientation {
+  Horizontal = 'horizontal',
+  Vertical = 'vertical',
+}
+
+type SeparatorProps = {
+  orientation: SeparatorOrientation;
```

Comment:
Orientation should be optional and default to horizontal.

We'll just have another prop called vertical, when it's passed the component becomes a vertical divider